### ISSUE TITLE
E-mail translation

### ISSUE THREAD

**Issue by fesarlis** [woothemes/woocommerce#51915]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

In my case:

1. I prefer to use English in my backend.
2. WP/WC are 100% translated (.po files).
3. E-mails are sent properly.

### Expected behavior

IMHO when there is translation present the mail message should get the hard-coded strings from the translation, not based on the language setting of the user reading it.

### Actual behavior

I have noticed, though, that even the translation exist, the email language matches the user language. This is unexpected.

So in my case, mails that are meant only for the shop manager (me) (eg. new order, cancelled order) are sent in English.

This also results in mixed language when using a mail customization plugin which will of course not interfere with core WC information (Order details, order number, product, price, quantity etc).

### Steps to reproduce

Force WC to send an email to the shop manager

### WordPress Environment

<details><summary>System Status Report</summary>

```
### WordPress Environment ###

WordPress address (URL): [Redacted]
Site address (URL): [Redacted]
WC Version: 9.3.3
Legacy REST API Package Version: The Legacy REST API plugin is not installed on this site.
Action Scheduler Version: ✔ 3.8.1
Log Directory Writable: ✔
WP Version: 6.6.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: –
Language: el
External object cache: ✔

### Server Environment ###

Server Info: 
PHP Version: 8.2.24
PHP Post Max Size: 25 MB
PHP Time Limit: 600
PHP Max Input Vars: 10000
cURL Version: 7.76.1
OpenSSL/3.0.7

SUHOSIN Installed: –
MySQL Version: 10.6.19-MariaDB-log
Max Upload Size: 25 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

[REDACTED]

### Post Type Counts ###

ame_ac_changeset: 12
attachment: 1295
blocks: 6
custom_css: 1
customize_changeset: 75
filter-field: 6
filter-seo-rule: 4
filter-set: 1
itsec-dash-card: 15
itsec-dashboard: 2
nav_menu_item: 155
oembed_cache: 4
page: 11
product: 1112
product_variation: 81
redirect_rule: 6
revision: 9
shop_order: 1
snitch: 1054
tm_global_cp: 2
view: 11
view-template: 8
wp_global_styles: 1
wp_navigation: 1
wp-types-group: 3
wp-types-term-group: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (42) ###

Admin Columns Pro: by AdminColumns.com – 6.4.12
The SEO Framework: by The SEO Framework Team – 5.0.6
Breadcrumb NavXT: by John Havlik – 7.3.1
Custom Post Type Cleanup: by keesiemeijer – 1.3.2
Database Cleaner (Pro): by Jordy Meow – 1.1.3
Disable Author Archives: by freemp – 1.3.5
Disable Comments: by WPDeveloper – 2.4.6
Disable Bloat for WordPress & WooCommerce PRO: by Disable Bloat – 3.4.4
Login/Signup Popup: by XootiX – 2.7.8
Payment Gateway Plugin for PayPal WooCommerce ( Free ): by WebToffee – 1.8.6
Filter Everything PRO: by Andrii Stepasiuk – 1.8.6
Flexible Shipping PRO: by Octolize – 2.19.0
Flexible Shipping: by Octolize – 4.25.8
Font Awesome: by Font Awesome – 4.5.0
Force Regenerate Thumbnails: by Exactly WWW – 2.2.0
Free Shipping Label: by Devnet – 3.1.2
HTML Editor Syntax Highlighter: by James Bradford – 2.4.4
Independent Analytics Pro: by Independent Analytics – 2.8.8
Solid Security Pro: by SolidWP – 8.5.0
Kadence WooCommerce Email Designer: by Kadence WP – 1.5.13
Loco Translate: by Tim Whitlock – 2.6.11
Log cleaner for Solid Security: by Rocket Apps – 1.4.1
Quform: by ThemeCatcher – 2.20.0
Safe Redirect Manager: by 10up – 2.2.0
Smart Attachment Page Remove: by Peter Raschendorfer – 4.0.5
SMTP Mailer: by naa986 – 1.1.15
Snitch: by pluginkollektiv – 1.2.0
Easy Updates Manager Premium: by Easy Updates Manager Team – 9.0.18
Toolset Types: by OnTheGoSystems – 3.5.2
UberMenu 3 - The Ultimate WordPress Mega Menu: by Chris Mavricos
SevenSpark – 3.8.5

W3 Total Cache: by BoldGrid – 2.7.6
Widget Options: by Widget Options Team – 4.0.7
WooCommerce Added To Cart Popup: by WeCreativez – 1.4.1
Checkout Field Editor for WooCommerce (Pro): by ThemeHigh – 3.6.3
Extra Product Sorting Options for WooCommerce: by SkyVerge – 2.10.0
Extra Product Options & Add-Ons for WooCommerce: by ThemeComplete – 6.4.6
Toolset WooCommerce Blocks: by OnTheGoSystems – 3.1.6
WooCommerce: by Automattic – 9.3.3
WP All Import Pro: by Soflyy – 4.8.0
Smush Pro: by WPMU DEV – 3.16.8
Toolset Views: by OnTheGoSystems – 3.6.16
WP All Import - WooCommerce Import Add-On Pro: by Soflyy – 3.3.1

### Inactive Plugins (12) ###

Admin Menu Editor Pro: by Janis Elsts – 2.26
AME Branding Add-on: by Janis Elsts – 1.3.7
AME Toolbar Editor: by Janis Elsts – 1.5.1
AutomateWoo: by WooCommerce – 6.1.0
Custom Product Tabs for WooCommerce: by Code Parrots – 1.8.5
Custom Product Tabs WP All Import Add-on: by YIKES
Inc. – 2.0.5

Disable Cart Fragments: by Optimocha – 2.3
Disable payment method / COD fees for Order for WooCommerce: by PI Websolution – 1.1.7.62
Query Monitor: by John Blackbourn – 3.16.4
User Role Editor Pro: by Vladimir Garagulia – 4.64.2
WP-Paginate: by Max Foundry – 2.2.2
WP Rocket: by WP Media – 3.17.0.2

### Dropin Plugins () ###

advanced-cache.php: advanced-cache.php
db.php: db.php
object-cache.php: object-cache.php

### Settings ###

Legacy API Enabled: –
Force SSL: –
Currency: EUR (€)
Currency Position: right
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: –

### Logging ###

Enabled: –
Handler: Automattic\WooCommerce\Internal\Admin\Logging\LogHandlerFileV2
Retention period: 30 days
Level threshold: –
Log directory size: 13 B

### WC Pages ###

Shop base: #474 - /e-shop/
Cart: #475 - /kalathi/ -  Contains the [woocommerce_cart] shortcode
Checkout: #476 - /tameio/ -  Contains the [woocommerce_checkout] shortcode
My account: #477 - /o-logariasmos-mou/
Terms and conditions: #301 - /oroi-xrisis/

### Theme ###

Name: Flatsome Child
Version: 3.0
Author URL: 
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 3.19.6
Parent Theme Author URL: https://uxthemes.com
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart-empty.php
flatsome/woocommerce/cart/cart-shipping.php
flatsome/woocommerce/cart/cart.php
flatsome/woocommerce/cart/mini-cart.php
flatsome-child/woocommerce/checkout/form-billing.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/checkout/form-coupon.php
flatsome-child/woocommerce/checkout/form-pay.php
flatsome/woocommerce/checkout/order-received.php
flatsome-child/woocommerce/checkout/payment.php
flatsome/woocommerce/checkout/terms.php
flatsome-child/woocommerce/checkout/thankyou.php
flatsome/woocommerce/content-product-cat.php
flatsome/woocommerce/content-product.php
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/content-widget-product.php
flatsome/woocommerce/global/breadcrumb.php
flatsome-child/woocommerce/global/quantity-input.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/header.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/rating.php
flatsome/woocommerce/loop/result-count.php
flatsome/woocommerce/loop/sale-flash.php
flatsome-child/woocommerce/myaccount/form-edit-account.php
flatsome/woocommerce/myaccount/form-login.php
flatsome/woocommerce/myaccount/navigation.php
flatsome-child/woocommerce/myaccount/orders.php
flatsome-child/woocommerce/notices/error.php
flatsome/woocommerce/notices/notice.php
flatsome-child/woocommerce/notices/success.php
flatsome/woocommerce/product-searchform.php
flatsome/woocommerce/single-product/photoswipe.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php
flatsome/woocommerce/single-product/product-thumbnails.php
flatsome/woocommerce/single-product/rating.php
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php


### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customize-store
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
pattern-toolkit-full-composability
product-custom-fields
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
printful
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page
launch-your-store

Disabled Features: experimental-blocks
minified-js
product-pre-publish-modal
settings
async-product-editor-category-field
product-editor-template-system
blueprint
reactify-classic-payments-settings

Daily Cron: ✔ Next scheduled: 2024-10-01 10:20:26 +03:00
Options: ✔
Notes: 67
Onboarding: completed

### Action Scheduler ###

Canceled: 25
Oldest: 2024-09-28 17:24:38 +0000
Newest: 2024-09-30 21:21:24 +0000

Complete: 1,223
Oldest: 2024-09-23 16:43:47 +0000
Newest: 2024-10-04 13:42:37 +0000

Failed: 41
Oldest: 2024-09-27 11:08:52 +0000
Newest: 2024-10-02 17:54:26 +0000

Pending: 1
Oldest: 2024-10-04 17:20:34 +0000
Newest: 2024-10-04 17:20:34 +0000


### Status report information ###

Generated at: 2024-10-04 16:54:11 +03:00
```
</details>

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by millerf** [comment]
@fesarlis Could you help me understand the issue you are having?

> I have noticed, though, that even the translation exist, the email language matches the user language. This is unexpected.
> So in my case, mails that are meant only for the shop manager (me) (eg. new order, cancelled order) are sent in English.

I am correct to understand that when an email is sent to the shop owner/manager, it is sent in English, independently of the user language choice?

**Comment by fesarlis** [comment]
No.
It is sent in English because the user language choice is English.

It is my understanding that an option should be present. For the least it should match the site language, not the user language.

**Comment by millerf** [comment]
Thanks for this explanation.

This is a duplicate of #36812

**Comment by albarin** [comment]
Closing as a duplicate https://github.com/woocommerce/woocommerce/issues/36812
